### DISCUSSION TITLE
Question about how collateral works

### DISCUSSION THREAD

**Discussion by mikekochh** [Cyfrin/foundry-full-course-cu#560] [Q&A]
Something that I thought about while working through Lesson 12:

Let's say someone puts up $100 ETH of collateral and then receives $30 worth of DSC. Then, this person goes and buys an NFT worth $30 with the DSC that they received. As time goes on, there collateral position is compromised and they are liquidated. They lose their collateral, but then what happens to the DSC that was used to buy and NFT? Does the person who sold the NFT lose the DSC?

Or, is it even possible to spend stablecoins like this? In that case, I don't understand what the point of opening this position would be if you can't even use the capital that you are borrowing. Any help would be greatly appreciated, thank you.

**Comment by Niferu** [comment]
Hello @mikekochh

It all depends on DeFi platform you are using, but it is possible in this case, once you got liquidated NFT would go back to lender as his interest from you.

The main purposes of borrowing coins are:

1. Trading and Investments: Borrowers might use the borrowed funds to trade other cryptocurrencies or invest in various DeFi protocols, seeking to generate higher returns than the cost of borrowing.

2. Short-Term Funding: Borrowers may need short-term funding for personal or business reasons, such as covering expenses or bridging financial gaps.

3. Leveraged Positions: Traders might use borrowed funds to take leveraged positions in the market, increasing their exposure to price movements.

4. Arbitrage and Market Making: Traders and liquidity providers may borrow funds to exploit price differences across different platforms or to provide liquidity on decentralized exchanges.

5. Hedging: Borrowers might use borrowed funds to hedge their positions or protect against potential losses in other investments.

I'm not the expert of DeFi, but I do have pretty vast knowledge about regular finances, so in case someone experienced in DeFi has other point of view, I would be very happy to see such person response here.

**Comment by theirrationalone** [comment]
# This can be a picture maybe... 

As time aged... You got liquidated... 

The liquidator will receive the bonus... And if the liquidating amount wasn't enough to pay bonus then this could/should be further Settlement... 

For example,at this condition: debts = $30, collateral amount = $30.

Network should pay bonus to liquidator and... 
That under collateralized account should be marked as lien account..... And all amount should be lien by protocol(DSCEngine) and Whenever in future that liquidated account gets funded then all its funds should be freezed against all charges & interests. 

A Lien amount is the locked or frozen amount available in your account that is not accessible for a specific time frame.

And haven't you noticed that liquidator already burned🔥debts on behalf of under collateralized account.

And it's totally depends on protocol And its constraints whether to forfeit their assets Or liquids or investments(possible in defi) etc Or not. 

And that would not be third person's business (that nft seller).
